Java/Ch 04. Trees and Graphs/Q4_08_First_Common_Ancestor/QuestionEBad.java

```diff
@@ -33,9 +33,9 @@ public static TreeNode commonAncestorBad(TreeNode root, TreeNode p, TreeNode q)
 	public static void main(String[] args) {
 		int[] array = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
 		TreeNode root = TreeNode.createMinimalBST(array);
-		TreeNode n3 = root.find(9);
-		TreeNode n7 = new TreeNode(6);//root.find(10);
-		TreeNode ancestor = commonAncestorBad(root, n3, n7);
+		TreeNode n9 = root.find(9);
+		TreeNode n6 = root.find(6);

```

Comment:
not sure why it was commented out in the original, I suppose it was an experiment